force-app/main/default/lwc/note/note.js

```diff
@@ -98,7 +65,7 @@ export default class Note extends LightningElement {
                             })
                         );
 
-                            console.log('Note added successfully');
+                            // console.log('Note added successfully');
                          await this.loadSavedData();
                         
                     }else {
```

Comment:
It is not possible you can reach this line because all errors you catch in "catch" statement in row 82